### PR TITLE
refresh vesting balance after withdraw

### PR DESCRIPTION
**Pull request by lanhaoxiang**
Refresh vesting balance after withdraw

### CODE REVIEW

**Comment by svk31** [comment]
This looks ok but are you sure it's necessary? The vesting balance objects should be updated automatically as the new objects are pushed by the backend.

**Comment by lanhaoxiang** [comment]
Yes, It's better to bind the state to chainstore, but I find vesting_balances were transformed into id array in Chainstore.js, what's the reason to do this?

File: app/stores/BlockchainStore.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ class BlockchainStore {
 
     onGetBlock(block) {
         if (!this.blocks.get(block.id)) {
+            if (!/Z$/.test(block.timestamp)) {
+                block.timestamp + 'Z';
```
Comment (svk31) [inline app/stores/BlockchainStore.js]: This needs a `+=` or `block.timestamp = block.timestamp + 'Z'` to actually change the timestamp.

**Comment by lanhaoxiang** [comment]
Should we close this PR, and looking for a better solution later?

**Comment by svk31** [comment]
I'm just not sure if it's needed, can you explain the actual bug you're trying to fix?

**Comment by lanhaoxiang** [comment]
I think it's not a bug, just an enhancement, I am trying to refresh the account balance after the user claimed the vesting balance. This make the user feel better after him/her made an operation.

**Pull request merged**